src/js/components/channels/viber-channel-content.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component, PropTypes } from 'react';
+import { LoadingComponent } from '../../components/loading';
+import { fetchViberQRCode } from '../../services/integrations-service';
+
+
+export class ViberChannelContent extends Component {
+    static contextTypes = {
+        ui: PropTypes.object
+    };
+
+    static propTypes = {
+        channelState: PropTypes.object.isRequired
+    };
+
+    render() {
+        const {channelState = {}} = this.props;
```

Comment:
That prop is required anyway, there's no need have a default. I've added the propTypes and remove the defaultProps.